### ISSUE TITLE
Allow custom PeeWees

### ISSUE THREAD

**Issue by DeinFreund** [springlobby/springlobby#928]
I searched my entire pc for `large PeeWee` only to find out the slapping is hardcoded. Please make it configurable, so I can slap others with Pancakes instead.

**Comment by silentwings** [comment]
you can use "/me slaps XXX around with a large XXX" or send a raw SAYEX message

**Comment by DeinFreund** [comment]
Yes, but that's rather impractical if you want to slap multiple people regularly. Like the s44 players who like to slap people around with Panzerkampfwagen.
